distributed/system_monitor.py

```diff
@@ -114,12 +114,11 @@ def __init__(
                 self.monitor_gil_contention = False
             else:
                 self.quantities["gil_contention"] = deque(maxlen=maxlen)
-                self._cumulative_gil_contention = 0.0
+                self.cumulative_gil_contention = 0.0
                 raw_interval = dask.config.get(
                     "distributed.admin.system-monitor.gil.interval",
                 )
-                interval = parse_timedelta(raw_interval, default="us") * 1e6
-
+                interval = parse_timedelta(raw_interval) * 1e6
```

Comment:
Don't interpret a raw `0.001` in the config (which violates the jsonschema) as 1us. Crash instead.